modules/bezug_carlogavazzilan/main.sh

```diff
@@ -1,5 +1,24 @@
 #!/bin/bash
+OPENWBBASEDIR=$(cd `dirname $0`/../../ && pwd)
+RAMDISKDIR="${OPENWBBASEDIR}/ramdisk"
+MODULEDIR=$(cd `dirname $0` && pwd)
+#DMOD="EVU"
+DMOD="MAIN"
+Debug=$debug
 
-python /var/www/html/openWB/modules/bezug_carlogavazzilan/readgavazzi.py $bezug1_ip
-wattbezug=$(</var/www/html/openWB/ramdisk/wattbezug)
-echo $wattbezug
+#For development only
+#Debug=1
+
+if [ ${DMOD} == "MAIN" ]; then
+	MYLOGFILE="${RAMDISKDIR}/openWB.log"
+else
+	MYLOGFILE="${RAMDISKDIR}/evu_carlo_gavazzi.log"
+fi
```

Comment:
`DMOD` wurde gerade erst in Zeile 6 gesetzt und dazwischen passiert auch nicht, bei dem sich die Variable nochmal ändern könnte. Entsprechend müsste hier der `else`-Zwei toter Code sein und könnte weg.